README.md

```diff
@@ -61,9 +117,11 @@ $ yarn dev
 - [Vue Storefront Middleware](https://docs.vuestorefront.io/v2/architecture/server-middleware.html).
 - Maximize lighthouse score.
```

Comment:
What is this about?

Reply:
Just that you have a good score in the lighthouse analysis.